infra/modules/bootstrap/main.tf

```diff
@@ -79,7 +65,8 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "tf_state" {
   bucket = aws_s3_bucket.tf_state.id
   rule {
     apply_server_side_encryption_by_default {
-      sse_algorithm = "AES256"
+      kms_master_key_id = aws_kms_key.tf_backend.arn
```

Comment:
can we just reuse the same kms key for all the terraform backend resources? since the whole point of all of these resources are the same – to set up the terraform backend and protect the tfstate files – it seems reasonable to me. the terraform lock table and tfstate bucket both will have tfstate data to protect. i'm less sure about the tf log bucket but it seemed excessive to have a separate kms key just for tf logs.

Reply:
that seems okay 🤷